### DISCUSSION TITLE
Will C/C++ compilation target be included?

### DISCUSSION THREAD

**Discussion by 0zl** [gleam-lang/gleam#2662] [Questions & support] [answered]
The title sums it up. Gleam is a great language with a clear syntax and a gentle learning curve. But it seems that Gleam is mainly designed for backend/server applications.

Is there any plan to support C/C++ as a compilation target? It would be awesome to create simple games with Gleam's type-safe features and interact with OpenGL libraries or Raylib.

Thanks!

**Comment by lpil** [comment, marked as answer]
Hello! No particular plans for any other backend at present, but long term we are interested in a native or low level VM target for some uses such as in-compiler code execution.
